chandra_aca/tests/test_maude_decom.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import os
+import pickle
+import numpy as np
+from astropy.table import vstack
+
+from chandra_aca import maude_decom
+
+
+with open(os.path.join(os.path.dirname(__file__), 'data', 'maude_decom.pcl'), 'rb') as f:
```

Comment:
I think Tom tends to use 'pkl' for pickle files but I don't know about the various conventions here.